tests/tests_no_auto_renew.yml

```diff
@@ -24,6 +27,7 @@
         subject_alt_name:
           - DNS:www.example.com
         auto_renew: no
+      - path: /tmp/defaultcert
```

Comment:
You need to add the `.crt` in the path. It would be also good to verify key-path and if `auto_renew: yes`